pkg/eve/Dockerfile.in

```diff
@@ -1,6 +1,6 @@
-FROM alpine:3.12 as tools
+FROM alpine:3.13.0 as tools
```

Comment:
Can you please drop this change from the PR @dkionka ?

Reply:
Done.